test/src/org/apache/jmeter/resources/PackageTest.java

```diff
@@ -136,9 +136,9 @@ private int readRF(String res, List<String> l) throws Exception {
                      * parameters and check if there is a { in the output. A bit
                      * crude, but should be enough for now.
                      */
-                    if (val.indexOf("{0}") > 0 && val.indexOf('\'') > 0) {
+                    if (val.contains("{0}") && val.contains("'")) {
```

Comment:
This change isn't exactly the same as the existing code, however it matches the comment, I think the > instead of >= in the original was a mistake.